src/amo/components/LandingPage.js

```diff
@@ -0,0 +1,118 @@
+import classNames from 'classnames';
+import React, { PropTypes } from 'react';
+import { compose } from 'redux';
+import { asyncConnect } from 'redux-connect';
+import { connect } from 'react-redux';
+
+import AddonsCard from 'amo/components/AddonsCard';
+import Link from 'amo/components/Link';
+import { loadLandingAddons } from 'amo/utils';
+import translate from 'core/i18n/translate';
+
+import './LandingPage.scss';
+
+
+export class LandingPageBase extends React.Component {
+  static propTypes = {
+    addonType: PropTypes.string.isRequired,
+    featuredAddons: PropTypes.array,
+    highlyRatedAddons: PropTypes.array,
+    popularAddons: PropTypes.array,
+    i18n: PropTypes.object.isRequired,
+  }
+
+  contentForType(addonType) {
+    const { i18n } = this.props;
+
+    const contentForTypes = {
```

Comment:
I see a lot of repetition here, especially with all the `<Link>` tags that mostly look the same. I think the only difference between `themes` and `extensions` will be the text and destinations. It would be nice to DRY this up with that in mind. It could happen in the next path when you fill in the destinations though.